src/Controller/front/ProfileController.php

```diff
@@ -60,7 +65,7 @@ public function follow(User $user, Request $request ,NotificationService $notifi
             if ($user->getId() !== $this->getUser()->getId()){
 
                 $message = 'à commencer à vous suivre';
-                $url = $this->generateUrl('app_front_profile_show',['username' => $user->getUsername()]);
+                $url = $this->generateUrl('app_front_profile_show',['username' => $this->getUser()->getUsername()]);
```

Comment:
Pourquoi avoir changer le $user en $this->getUser() ?

Reply:
c'est pas lui c'est moi , psk c'est pour redirigez sur la page du profil de celui qui t'as follow et non ta propre page